docs-css/docs-sidebar.css

```diff
@@ -0,0 +1,79 @@
+.sidebar{
+    grid-area: sidebar;
+    position: sticky;
+    top: 0;
+    height: calc(100vh - 2rem);
+    overflow-y: auto;
+    min-width: 15rem;
+}
+.side-navbar{
+    display: flex;
+    flex-direction: column;
+    gap: 0.7rem;
+  }
+  .title{
+    margin-top: 1rem;
+  }
+  .subtitle{
+      color: #909396;
+      line-height: 1rem;
+  }
+  .drawer-menu{
+      height: 100%;
+      overflow-y: auto;
+  }
+  .nav-lists{
+      display: flex;
+      flex-direction: column;
+      justify-items: center;
+      gap: 0.3rem;
+      list-style: none;
+  }
```

Comment:
please do care of indentation here. please format this.

Reply:
Done